modules/vaos/app/services/vaos/v2/mobile_facility_service.rb

```diff
@@ -22,10 +22,10 @@ def get_facilities(query_params, pagination_params = {})
         end
       end
 
-      def get_scheduling_configurations(facility_ids, cc_enabled, pagination_params = {})
+      def get_scheduling_configurations(facility_ids, cc_enabled = nil, pagination_params = {})
         params = {
-          facility_ids: facility_ids,
-          cc_enabled: cc_enabled
+          facilityIds: facility_ids,
+          ccEnabled: cc_enabled
         }.merge(page_params(pagination_params)).compact
```

Comment:
why are we passing camelcase attributes? We have a middleware that converts snakecase to camelcase automatically. You should be able to work with snakecase everywhere in rails and just let the middleware do its thing.

Reply:
The original bug report was that the facility_ids query parameter sent to VAMF was not be sent as camel case (as required by VAMF). The changes were made to comply with the Mobile Facility Service swagger docs. 

Olive branch does not seem to change snake/camel case between vets website and vets api, but not between vets api and VAMF.

Reply:
ah, my bad I thought it was a request body. This makes sense... i was thinking the faraday middleware handles those, but it does not.